openwisp_users/tests/test_admin.py

```diff
@@ -375,3 +375,44 @@ def test_change_user(self):
         self.assertNotContains(response, 'error')
         user.refresh_from_db()
         self.assertEqual(user.bio, params['bio'])
+
+
+class TestMultitenantAdmin(TestMultitenantAdminMixin, TestOrganizationMixin):
+
+    """ Only OrganizationUserAdmin and OrganizationOwnerAdmin are are multitenant as of now,
+    therefore only they're being tested.
+    Change tests accordingly if more admin views are to be made multitenant. """
```

Comment:
What does `therefore only they're being tested`  mean?
I'd simply remove this docstring since it does not really add any value, can you remove it please?

Reply:
I'll get rid of it.